project/MimaExcludes.scala

```diff
@@ -82,6 +82,10 @@ object MimaExcludes {
             // SPARK-5166 Spark SQL API stabilization
             ProblemFilters.exclude[IncompatibleMethTypeProblem]("org.apache.spark.ml.Transformer.transform"),
             ProblemFilters.exclude[IncompatibleMethTypeProblem]("org.apache.spark.ml.Estimator.fit")
+          ) ++ Seq(
+            // SPARK-5315 Spark Streaming Java API returns Scala DStream
+            ProblemFilters.exclude[MissingMethodProblem](
+              "org.apache.spark.streaming.api.java.JavaDStreamLike.reduceByWindow")
```

Comment:
Hi all, I tested with a clean build, still Mima complains about it, seems it is false positive.

```
[error]  * method reduceByWindow(org.apache.spark.api.java.function.Function2,org.apache.spark.streaming.Duration,org.apache.spark.streaming.Duration)org.apache.spark.streaming.api.java.JavaDStream in trait org.apache.spark.streaming.api.java.JavaDStreamLike does not have a correspondent in old version
[error]    filter with: ProblemFilters.exclude[MissingMethodProblem]("org.apache.spark.streaming.api.java.JavaDStreamLike.reduceByWindow")
```